py2many/rewriters.py

```diff
@@ -145,10 +146,17 @@ def visit_If(self, node):
             if hasattr(node, "scopes") and len(node.scopes) > 1:
                 rename(node.scopes[-2], "main", "main_func")
             # ast.parse produces a Module object that needs to be destructured
-            ret = ast.parse("def main(argc: int, argv: List[str]): True").body[0]
+            if self.main_signature_arg_names == {"argc", "argv"}:
```

Comment:
I like this solution!

Reply:
(wrong PR)

btw, if you give me non-main branch access, I can create PRs which use a prior PR as a base, so that each PR doesnt include preliminary commits. c.f. https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request